tests/settings_tests/tests.py

```diff
@@ -445,3 +447,25 @@ def test_tuple_settings(self):
             finally:
                 del sys.modules['fake_settings_module']
                 delattr(settings_module, setting)
+
+
+class TestDatabaseMirror(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_without_mirror(self):
+        Simple.objects.using('default').create()
+        Simple.objects.using('other').create()
+
+        self.assertEqual(Simple.objects.count(), 2)
```

Comment:
Is this right? 

The two `create` calls create two objects in separate databases. The `count` call fetches from a single database. You need a `1` here no?